Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM golang:1.13 as bd
-RUN adduser --disabled-login appuser
+RUN adduser --disabled-login --gecos "" appuser
```

Comment:
@antweiss, nice. This makes much sense. Thanks for this.

Reply:
Opened https://github.com/layer5io/meshery-istio/pull/28 just for this change